pkg/server/middleware.go

```diff
@@ -43,13 +43,13 @@ func authMiddlewareWithUser(a *auth.Authenticator, handlerFunc func(user *auth.U
 		}
 		if !safe {
 			if err := a.VerifySourceOrigin(r); err != nil {
-				klog.Infof("invalid source origin: %v", err)
+				klog.V(4).Infof("invalid source origin: %v", err)
```

Comment:
Also not expected. We should surface this one.

Reply:
if this one is expected, then shouldn't we put it on higher log level ?